cache/class-vary-cache.php

```diff
@@ -211,14 +211,17 @@ private static function decrypt_cookie_value( $cookie_value ) {
 	 * @access  private
 	 */
 	private static function parse_group_cookie() {
-		if ( isset( $_COOKIE[ self::COOKIE_SEGMENT ] ) ) {
-			$cookie_value = sanitize_text_field( wp_unslash( $_COOKIE[ self::COOKIE_SEGMENT ] ) );
+		if ( isset( $_COOKIE[ self::COOKIE_SEGMENT ] ) || isset( $_COOKIE[ self::COOKIE_AUTH ] ) ) {
+
 			if ( self::is_encryption_enabled() ) {
+				$cookie_value = sanitize_text_field( wp_unslash( $_COOKIE[ self::COOKIE_AUTH ] ) );
```

Comment:
What's the reason for using `sanitize_text_field` and `wp_unslash` here? One concern would be that they strip or modify the cookie value in a way that breaks things so we should just make sure that they're actually needed. It might be better to sanitize further down when we actually parse the group information.

Reply:
Yeah, this might be overkill and you are right about the wp_unslash, especially since the backlash is one the characters it can contain (I had actually removed it at one point during testing)